OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -47,26 +52,16 @@
     <StartArguments>Game.Mod=ts</StartArguments>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Open.Nat">
-      <HintPath>..\thirdparty\download\Open.Nat.dll</HintPath>
-    </Reference>
-    <Reference Include="Eluant">
-      <HintPath>..\thirdparty\download\Eluant.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
-    <Reference Include="ICSharpCode.SharpZipLib">
-      <HintPath>..\thirdparty\download\ICSharpCode.SharpZipLib.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
-    <Reference Include="MaxMind.Db">
-      <HintPath>..\thirdparty\download\MaxMind.Db.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
     <ProjectReference Include="..\OpenRA.PostProcess\OpenRA.PostProcess.csproj">
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
     <None Include="App.config" />
+    <PackageReference Include="MaxMind.Db" Version="2.6.1" />
+    <PackageReference Include="Open.NAT.Core" Version="2.1.0.5" />
```

Comment:
This points at https://github.com/Aragas/Open.NAT.Core, which appears to be an abandoned fork of the original https://github.com/lontivero/Open.NAT.

If https://www.nuget.org/packages/Open.Nat/ isn't suitable we should IMO fork our own nuget package.